inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/DefaultMessageSender.java

```diff
@@ -194,7 +194,7 @@ public SendResult sendMessage(byte[] body, String attributes, String msgUUID,
     }
 
     public SendResult sendMessage(byte[] body, String groupId, String streamId, long dt, String msgUUID,
-            long timeout, TimeUnit timeUnit) {
+                                  long timeout, TimeUnit timeUnit) {
```

Comment:
Please don't change code style. Refer to https://github.com/apache/inlong/discussions/3082.